notebooks/features/hyperparameter_tuning/HyperOpt-SynapseML.ipynb

```diff
@@ -49,7 +54,31 @@
    "outputs": [],
    "source": [
     "# version >= 1.28.0 supports reading logModelAllowlistFile from url\n",
-    "%pip install mlflow==1.29.0"
+    "# %pip install mlflow==1.29.0\n",
+    "os.system(\"pip install mlflow==1.29.0\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from synapse.ml.core.platform import *\n",
+    "from pyspark.sql import SparkSession\n",
+    "\n",
+    "spark = SparkSession.builder.getOrCreate()\n",
+    "\n",
+    "if running_on_synapse_internal():\n",
+    "    experiment_name = \"hyperopt-synapseml\"\n",
+    "elif running_on_synapse():\n",
+    "    experiment_name = \"/Shared/hyperopt-synapseml\"\n",
```

Comment:
```suggestion
    "    experiment_name = \"hyperopt-synapseml\"\n",
```
Let's use hyperopt-synapseml as experiment name directly, only databricks internally has a constraint on the experiment name has to be an absolute path.